careers.md

```diff
@@ -17,4 +17,6 @@ One or more of the following would also be beneficial:
 - history of collaborating between different projects
 
 
-For more details and how to apply, please visit the [OMSF careers page](https://omsf.io/about/careers/).
+If you are a software scientist with a relevant interest in getting involved with open software development, please fill out the [application form](https://docs.google.com/forms/d/e/1FAIpQLSdv8B5mx_imKl0I0OOVs9zMjSHlAj2WuJPVXjqSVQpCLHW4YQ/viewform). Applications will be reviewed on a regular basis or considered when a new position opens.
+
+For more details, please visit the [OMSF careers page](https://omsf.io/about/careers/).
```

Comment:
I don't necessarily understand this, if the details of the career things are on the OMSF website, then shouldn't the form be there too? It feels weird to ask folks to look somewhere else for details and then provide the application form beforehand?

Reply:
Should we just delete the 'For more details' line?

Reply:
Or maybe we should just let omsf deal with applications???

Reply:
I'd rather it be here.

1. There's no reason to require 2 clicks from applicants.

2. OMSF's website is hard to navigate. Karmen knows this, and has mentioned that she wants to clean it up. Relevance here: the URL for the careers page might change. So I'd rather not point at it until (since we don't have broken links testing yet.) A broken link for a job applicant makes us look REAL bad.

We might change that in the future (adding broken link checking is good, and I have no problem framing this as applying to a job at OMSF as opposed to OpenFE, since that's what it really is -- although the text here would need to make that clear), but for now I think we should keep it here.

Reply:
Perfect!